### ISSUE TITLE
Handle cookies on deployment

### ISSUE THREAD

**Issue by colearendt** [rstudio/rsconnect-jupyter#173]
If deploying to a HA Connect environment, storing the "sticky" cookie is very important for the deployment process, lest the "task" search hit the wrong node.

Looking here: https://github.com/rstudio/rsconnect-jupyter/blob/0fa0f11f15879fca7a3db4faaf1e2c24e17fb84e/rsconnect_jupyter/api.py#L232-L250

Or maybe here: https://github.com/rstudio/rsconnect-jupyter/blob/0fa0f11f15879fca7a3db4faaf1e2c24e17fb84e/rsconnect_jupyter/api.py#L186-L189

It looks like `_update_cookie` should perhaps be called on the response object, but isn't? I could very much be missing where this is implemented, though!

We are seeing behavior that suggests the sticky cookie is not being stored by rsconnect-jupyter, which is causing problems in our HA Connect environment.

**Comment by colearendt** [comment]
The error from RSC - _super_ helpful for diagnosing!
```
Task lookup failures can indicate that a load balancer is not using sticky sessions or a client is not including the session cookie. Details: https://docs.rstudio.com/connect/admin/high-availability.html
```

**Comment by joncfoo** [comment]
brief discussion in slack thread: https://rstudio.slack.com/archives/CKDQZGR26/p1565725589069800

**Comment by mmarchetti** [comment]
Fixed in #178
